akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/ActorLoggingSpec.scala

```diff
@@ -158,6 +158,15 @@ class ActorLoggingSpec extends ActorTestKit with TypedAkkaSpec {
           ctx.log.error(marker, cause, "{} {} {} {}", "arg1", "arg2", "arg3", "arg4")
           ctx.log.error(marker, cause, "{} {} {} {} {}", Array("arg1", "arg2", "arg3", "arg4", "arg5"))
 
+          Logging.AllLogLevels.foreach(level ⇒ {
+            ctx.log.log(level, "message")
+            ctx.log.log(level, "{}", "arg1")
+            ctx.log.log(level, "{} {}", "arg1", "arg2")
+            ctx.log.log(level, "{} {} {}", "arg1", "arg2", "arg3")
+            ctx.log.log(level, "{} {} {} {}", "arg1", "arg2", "arg3", "arg4")
+            ctx.log.log(level, "{} {} {} {} {}", Array("arg1", "arg2", "arg3", "arg4", "arg5"))
+          })
+
```

Comment:
good, include tests for the marker one as well please: ` def log(level: LogLevel, marker: LogMarker, message: String): Unit`